src/foam/nanos/analytics/mixpanel/rules.jrl

```diff
@@ -0,0 +1,16 @@
+p({
```

Comment:
Not all FOAM/NANOS instances will want to do this so this would be better to be added on the NP side. Same with services.jrl file below. Or make the MixpanelAnalyticEventDAO a rule, which is easier to add in a new journal without having to replace the old entry.